### DISCUSSION TITLE
TELEGRAM VE DİSCORD ÖDÜLLERİ

### DISCUSSION THREAD

**Discussion by Musaix** [IBAX-io/go-ibax#1579] [Soru Cevap - Türkçe]
Arkadaşlar telegram ve discord katılımı sağladımız zaman ödül verileceği yazıyor. İkisine de katıldım ama bana ödül puanları gelmedi alabilen var mı ?

**Comment by Abdullahoda** [comment]
sordum acele etmeyin beklemede kalın dediler.

**Comment by Yusufknk** [comment]
Bunun için yöneticilerle görüşüyoruz. İlk başta form olarak yapacaklardı ama birden fazla bot hesap ihtimali olduğu için bu yöntemi elediler. Başka bir metot üzerinde çalışıyorlar.

**Comment by THECROVV** [comment]
Telegram ve discord a katılım sonucu puan vereceği bilgisi nerede geçmekte acaba, öyle bir bilgiye ulaşamadım
